<launch>

  <node name="pointcloud_UDP_server_node" pkg="serial_node" type="pointcloud_UDP_server_node">

    <param name="debug" value="true" />

    <param name="output_topic"               value="/new_cloud" />
    <param name="Output_Buffer_Length"       value="100" />

  </node>

</launch>
